documentation/certificates.md

```diff
@@ -0,0 +1,91 @@
+Streisand PKI
+=============
+
+Streisand includes a role responsible for PKI certificate generation,
+separated into two distinct tasks:
+
+ - CA/Server
+ - Client(s)
+
+The first of which will generate a certificate authority, and a server 
```

Comment:
nit: I think you can remove "of which" and just say "The first will generate"...